src/routes/hud/elements/notifications/Notification.svelte

```diff
@@ -21,21 +21,21 @@
             "a c";
     grid-template-columns: max-content 1fr;
     column-gap: 10px;
-    background: rgba(0, 0, 0, 0.4);
-    border-radius: 60px;
+    background: rgba(0, 0, 20, 0.4);
+    border-radius: 20px;
     width: 300px;
     overflow: hidden;
     padding: 10px;
     margin-bottom: 10px;
-    box-shadow: 0px 0px 4px 4px rgba(0, 0, 0, 0.4);
+    font-family: 'Product Sans';
```

Comment:
It would be better to stop declaring the font and for it to be declared in app.scss